src/m365/teams/commands/conversationmember/conversationmember-add.ts

```diff
@@ -0,0 +1,282 @@
+          }
+
+          return resolve(teamItem.id);
+        }, err => { reject(err) });
+    })
+  }
+
+  private getChannelId(teamId: string, args: CommandArgs): Promise<string> {
+    if (args.options.channelId) {
+      const channelIdRequestOptions: any = {
+        url: `${this.resource}/v1.0/teams/${encodeURIComponent(teamId)}/channels/${encodeURIComponent(args.options.channelId as string)}`,
+        headers: {
+          accept: 'application/json;odata.metadata=none'
+        },
+        responseType: 'json'
+      };
+
+      return new Promise<string>((resolve: (channelId: string) => void, reject: (error: any) => void): void => {
+        request
+          .get<Channel>(channelIdRequestOptions)
+          .then((response: Channel) => {
+            const channelItem: Channel | undefined = response;
+            return resolve(channelItem.id);
+          }, (err: any) => {
+            if (err.error && err.error.code == "NotFound") {
+              return reject(`The specified channel '${args.options.channelId}' does not exist or is invalid in the Microsoft Teams team with ID '${teamId}'`);
+            } else {
+              return reject(err);
+            }
+          });
+      });
+    } else {
+      const channelRequestOptions: any = {
+        url: `${this.resource}/v1.0/teams/${encodeURIComponent(teamId)}/channels?$filter=displayName eq '${encodeURIComponent(args.options.channelName as string)}'`,
+        headers: {
+          accept: 'application/json;odata.metadata=none'
+        },
+        responseType: 'json'
+      };
+
+      return new Promise<string>((resolve: (channelId: string) => void, reject: (error: any) => void): void => {
+        request
+          .get<{ value: Channel[] }>(channelRequestOptions)
+          .then(response => {
+            const channelItem: Channel | undefined = response.value[0];
+
+            if (!channelItem) {
+              return reject(`The specified channel '${args.options.channelName}' does not exist in the Microsoft Teams team with ID '${teamId}'`);
+            }
+
+            return resolve(channelItem.id);
+          }, err => reject(err));
+      });
+    }
+  }
+
+  private getUserId(args: CommandArgs): Promise<string[]> {
+    if (args.options.userId) {
+      return Promise.resolve(args.options.userId.split(',').map(u => u.trim()));
+    }
+
+    return new Promise<string[]>((resolve: (userId: string[]) => void, reject: (error: any) => void): any => {
+      const getSingleUser: (userDisplayName: string) => Promise<string> = (userDisplayName: string): Promise<string> => {
+        const userRequestOptions: any = {
+          url: `${this.resource}/v1.0/users?$filter=displayName eq '${encodeURIComponent(userDisplayName as string)}'`,
+          headers: {
+            'content-type': 'application/json'
```

Comment:
I don't think we need to specify here the `content-type` header as it's a GET request